package.json

```diff
@@ -22,7 +22,9 @@
     "codegen:check": "npm run codegen && git diff --exit-code",
     "lint": "eslint . --ext .ts",
     "error-code-doc": "node ./internals-js/dist/genErrorCodeDoc.js > ./docs/source/errors.md",
-    "error-code-doc:check": "npm run error-code-doc && git diff --exit-code"
+    "error-code-doc:check": "npm run error-code-doc && git diff --exit-code",
+    "hints-doc": "node ./composition-js/dist/genHintDoc.js > ./docs/source/hints.md",
+    "hints-doc:check": "npm run hints-doc && git diff --exit-code"
```

Comment:
**suggestion** Should we add this to `.circleci/config.yaml` so we run checks for the docs and the hints? (obviously pending any refactors to combine errors/hints docs)

```
npm run error-code-doc:check hints-doc:check
```

Reply:
I'd say write a ticket for that, just in case there is weirdness ahead of release and it becomes a flaky test for the next two days

Reply:
With permission from @cpeacock, I did added the check, since it's been pretty useful for the error doc and hasn't been flaky, so I'm somewhat confident about.

I did added it as a separate check, mostly because
```
npm run error-code-doc:check hints-doc:check
```
somehow didn't work locally for me, but each worked separately, so I didn't wanted to risk screwup up something.

Having a different check is admittedly a tad wasteful, though it's also a tad more informative (we'll probably merge both doc eventually and end up with a single check anyway).